e2e/tests/workspace/workspace-ui.spec.ts

```diff
@@ -0,0 +1,105 @@
+
+  let page;
+
+  beforeEach(async () => {
+    page = await Chrome.setup();
+  });
+
+  afterEach(async () => {
+    await Chrome.teardown();
+  });
+
+  test('Can Create new workspace from Home page', async () => {
+    const home = new Home(page);
+    const link = await home.getCreateNewWorkspaceLink();
+    expect(link).toBeTruthy();
+    await link.click();
+
+    // check page title
+    const actPageTitle = await page.title();
+    expect(actPageTitle).toMatch('Create Workspace');
+
+    const workspace = new Workspaces(page);
+    // check Workspace-Name textfield
+    expect(await workspace.input_workspace_name()).toBeTruthy();
+    // check dataset select options
+    expect(await workspace.select_dataSet()).toBeTruthy();
+  }, 60 * 1000);
+
+
+  test('Create new workspace page', async () => {
+    const workspaces = new Workspaces(page);
+    await workspaces.goURL();
+    await workspaces.click_button_CreateNewWorkspace();
+
+    // check page title
+    const actPageTitle = await page.title();
+    expect(actPageTitle).toMatch('Create Workspace');
+
+    const workspace = new Workspaces(page);
+
+    // webelement Disease-focused research: out-of-box
+    const elementDiseaseName = await workspace.element_diseaseName();
+    const diseaseNameCheckbox = await elementDiseaseName.checkbox();
+    expect(diseaseNameCheckbox).toBeTruthy();
+    expect(await getProperty(page, diseaseNameCheckbox, 'checked')).toBeFalsy();
+    expect(await getProperty(page, diseaseNameCheckbox, 'disabled')).toBeFalsy();
+
+    const diseaseNameTextInput = await elementDiseaseName.textfield();
+    expect(diseaseNameTextInput).toBeTruthy();
+    expect(await getProperty(page, diseaseNameTextInput, 'disabled')).toBeTruthy();
+
+    // await (diseaseNameCheckbox).click(); // click checkbox to change
+    await page.evaluate(elem => elem.click(), diseaseNameCheckbox);
+    // TODO wait async for checked and disabled checking or test will fail
+    await page.waitFor(1000);
+    expect(await getProperty(page, await elementDiseaseName.checkbox(), 'checked')).toBeTruthy();
+    expect(await getProperty(page, await elementDiseaseName.checkbox(), 'disabled')).toBeFalsy();
+
+
+    // check Workspace Name textfield
+    expect(await workspace.input_workspace_name()).toBeTruthy();
+
+    // check dataset select option
+    expect(await workspace.select_dataSet()).toBeTruthy();
+
+    // check all fields in Question #1. What is the primary purpose of your project?
+    expect(await workspace.element_question1_populationHealth().checkbox()).toBeTruthy();
+    expect(await workspace.element_question1_MethodsDevelopment().checkbox()).toBeTruthy();
+    expect(await workspace.element_question1_DrugTherapeuticsDevelopment().checkbox()).toBeTruthy();
+    expect(await workspace.element_question1_ResearchControl().checkbox()).toBeTruthy();
+    expect(await workspace.element_question1_EducationalPurpose().checkbox()).toBeTruthy();
```

Comment:
there's a For Profit Purpose box in here as well

Reply:
done